bubblesub/cmd/save_audio_sample.py

```diff
@@ -35,6 +36,9 @@ def is_enabled(self) -> bool:
         return (
             self.api.audio.current_stream
             and self.api.audio.current_stream.is_ready
+        ) and (
+            self.api.gui.current_view == ViewState.Audio
+            or self.api.gui.current_view == ViewState.Full
```

Comment:
I should be able to save audio for the current line even if the  spectrogram is hidden